### PR TITLE
SQL DataNode doc for Microsoft SQL Server

### PR DESCRIPTION
**Pull request by florian-vuillemot**
#162

### CODE REVIEW

**Comment by florian-vuillemot** [comment]
@jrobinAV I would like to delete the Docker documentation for Airflow. It is in the Git history. What do you think about it?

File: docs/manuals/deployment/docker/docker.md [hunk 1]
```diff
@@ -28,6 +26,17 @@ RUN pip install -r requirements.txt
 CMD python my-app.py
 ```
 
+!!! Note
+
+    If you are using a Microsoft SQL Server as `SQLDataNode^`, you need to add the following commands
```
Comment (jrobinAV) [inline docs/manuals/deployment/docker/docker.md]:
```suggestion
    If you are using a `SQLDataNode^` with a Microsoft SQL Server, you need to add the following commands
```

File: docs/manuals/deployment/linux/redhat.md [hunk 2]
```diff
@@ -44,6 +44,12 @@ sudo mv `pwd`/.local/bin/uwsgi /usr/bin/uwsgi
 sudo restorecon /usr/bin/uwsgi
 ```
 
+!!! Note
+
+    If you are using a Microsoft SQL Server as `SQLDataNode^`, you need to install your corresponding
```
Comment (jrobinAV) [inline docs/manuals/deployment/linux/redhat.md]:
```suggestion
    If you are using a `SQLDataNode^` with a Microsoft SQL Server, you need to install your corresponding
```

File: docs/manuals/deployment/linux/ubuntu.md [hunk 3]
```diff
@@ -27,6 +27,12 @@ sudo pip install uwsgi gevent
 sudo ln -s `pwd`/.local/bin/uwsgi /usr/bin/uwsgi
 ```
 
+!!! Note
+
+    If you are using a Microsoft SQL Server as `SQLDataNode^`, you need to install your corresponding
```
Comment (jrobinAV) [inline docs/manuals/deployment/linux/ubuntu.md]:
```suggestion
    If you are using a `SQLDataNode^` with a Microsoft SQL Server, you need to install your corresponding
```

**Review by jrobinAV — APPROVED** [review]
I just proposed some rephrasing that sounds more English to me.

**Comment by jrobinAV** [comment]
> @jrobinAV I would like to delete the Docker documentation for Airflow. It is in the Git history. What do you think about it?

@florian-vuillemot 
OK.
I believe it is not exposed anymore, but we can delete it.

**Pull request merged**